core_backend/app/utils.py

```diff
@@ -47,3 +49,47 @@ def setup_logger(
     logger.addHandler(handler)
 
     return logger
+
+
+class HttpClient:
+    """
+    HTTP client for call other endpoints
+    """
+
+    session: aiohttp.ClientSession | None = None
+
+    def start(self) -> None:
+        """
+        Create AIOHTTP session
+        """
+        self.session = aiohttp.ClientSession()
+
+    async def stop(self) -> None:
+        """
+        Close AIOHTTP session
+        """
+        if self.session is not None:
+            await self.session.close()
+        self.session = None
+
+    def __call__(self) -> aiohttp.ClientSession:
+        """
+        Get AIOHTTP session
+        """
+        assert self.session is not None
+        return self.session
+
+
+_HTTP_CLIENT: aiohttp.ClientSession | None = None
```

Comment:
As per aiohttp docs. We don't want to create a new session for each request. This ensures we are just using one global ClientSession